packages/unity-interface/portableExperiencesUtils.ts

```diff
@@ -127,6 +128,10 @@ export async function declareWantedPortableExperiences(pxs: StorePortableExperie
     }
   }
 
+  // TODO: this is an ugh workaround, fix controlling the scene lifecycle
+  // knowing when the scene was completly removed and then re-spawn it
+  await sleep(250)
```

Comment:
this sounds super arbitrary.
Can we attach the dispose to a promise so we can wait to all the workers to be removed ?  Is it possibe ?
```ts 
await forceStopSceneWorker(scene.worker))

```